src/main/java/org/example/repository/StudentFeeRepository.java

```diff
@@ -25,7 +29,42 @@ public static StudentFeeRepository getInstance() {
     // 레포에 들어있는 csv 파일을 읽어와서 StudentInfo 객체를 만들고 HashMap에 저장하는 메소드
     // 필요할 경우 private method를 추가하여 구현
     public void initStudentInfo() {
-
+        try {
+            // 엑셀 파일 읽기
+            FileInputStream fis = new FileInputStream("23.xlsx");
```

Comment:
```suggestion
            FileInputStream inputStream = new FileInputStream("23.xlsx");
```

fis처럼 줄임말을 사용하는 것은 지양해주세요. 현재는 해당 inputStream이 중요한 부분은 아니라서 InputStream 정도로 적어도 괜찮을 것 같습니다.

Reply:
fis -> fileInputStream 수정했습니다.